tsdef/Track.d.ts

```diff
@@ -0,0 +1,13 @@
+import { EventEmitter } from 'events';
+
+export namespace Track {
+  type Kind = 'audio' | 'video' | 'data';
```

Comment:
```suggestion
  enum Kind {
    audio = 'audio',
    data = 'data',
    video = 'video'
  }
```

Reply:
@manjeshbhargav After chatting with @timmydoza a bit to go over the definitions. We learned that you actually cannot have enums inside of definition files because the compiler does not compile the definitions into anything. So if we tried to use `Kind`, it would come back as undefined. I think we should keep them as types since there actually is a drop down menu when I re-tried today!

This is what it would look like trying to access LogLevel as a `type`:
![image](https://user-images.githubusercontent.com/43423318/100291840-b1ef0000-2f33-11eb-9c17-aa8dc6675fab.png)

Which still has a drop down menu to select which `LogLevel` we want!

Reply:
Correct. In an ordinary .ts file, an enum would be compiled into a JavaScript object. But an enum inside a .d.ts file never gets compiled into an object, so the compiled JS file will try to `require()` an object that doesn't exist.